tests/test_create.py

```diff
@@ -35,18 +43,29 @@ def setUp(self):
     #     print(yaml.dump(settings.default_settings, default_flow_style=False))
     #     print()
 
-    def test_create_instance(self):
-        results = self.test_instance.create()
-        # print(yaml.dump(results, default_flow_style=False))
-        self.assertTrue(results["created"])
+    def test_load_instance_create(self):
+        ''' Create a fresh instance
+
+        Testing if instance directory is create on load
+        '''
+        self.test_instance.load()
+        result = os.path.exists(self.instance_path)
+        self.test_instance.destroy()
+        self.assertTrue(result)
 
-    def test_create_instance_exists(self):
-        results = self.test_instance.create()
-        self.assertFalse(results["created"])
+    def test_load_instance_exists(self):
+        os.makedirs(self.instance_path + "/bucket1")
+        with open(self.instance_path + "/bucket1/" + "file1.yml", "w") as outfile:
```

Comment:
the with syntax should only be used if there's an embedded try/except etc. 

this is more simply written as: 
```outfile = open(self.instance_path + '/bucket1/' + 'file1.yml', 'w')```